### DISCUSSION TITLE
IWP ITEM CREATOR

### DISCUSSION THREAD

**Discussion by cho66x** [naverz/zepeto-studio-global#42] [General]
How to buy item creator in the world using events ui?

**Comment by jhslee** [comment]
Hey @anggacho 

I believe you're talking about linking an IWP purchase even through UI? You can create an IWP item through the console first from within your world, please ([see the guide here](https://studio.zepeto.me/guides/making-world-products)). 

1. Create an IWP product. 
2. Make a note of the content item id, and create a unity Canvas button. 
3. Attach the IWP buttons typescript, and follow the guide [here](https://studio.zepeto.me/guides/making-world-products) 
4. Watch the IWP even in action!!

**Comment by cho66x** [comment]
Thanks for the explanation @jhslee
